### PR TITLE
[#IOPID-1908 #IOPID-1920] Add change feed processor for profiles container

### PR DESCRIPTION
**Pull request by BurnedMarshal**
#### List of Changes

Add new change feed cosmos db trigger on the profile container.
Add new configuration env to bind the kafka queue for outbound documents.
PDV enrich utils support multiple document type

#### Motivation and Context

To export profiles data to PDND we add a new processor for profiles documents

#### How Has This Been Tested?

unit test

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by gquadrati — CHANGES_REQUESTED** [review]
LGTM, just some small comments and we are ready to test it.

File: utils/formatter/profilesAvroFormatter.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+): Omit<profile, "schema" | "subject"> => ({
+  id: retrievedProfilesWithPdvId.id.replace(
+    retrievedProfilesWithPdvId.fiscalCode,
+    retrievedProfilesWithPdvId.userPDVId ?? "UNDEFINED"
+  ),
+  userPDVId: retrievedProfilesWithPdvId.userPDVId ?? "UNDEFINED",
+  // eslint-disable-next-line no-underscore-dangle
+  timestamp: retrievedProfilesWithPdvId._ts * 1000,
+  acceptedTosVersion:
+    retrievedProfilesWithPdvId.acceptedTosVersion?.toString() ?? "UNDEFINED",
```
[the remark this answers is not in the document]
Comment (BurnedMarshal) [reply utils/formatter/profilesAvroFormatter.ts]: Fixed in [b9ac886](https://github.com/pagopa/io-functions-elt/pull/84/commits/b9ac8862622acc1e7cdf57071bf61b0736d4ccd3)

File: utils/formatter/profilesAvroFormatter.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  blockedInboxOrChannels:
+    (retrievedProfilesWithPdvId.blockedInboxOrChannels as {
+      // eslint-disable-next-line functional/prefer-readonly-type
+      [index: string]: BlockedInboxOrChannelEnum[];
+    }) ?? {},
+  isEmailEnabled: retrievedProfilesWithPdvId.isEmailEnabled ?? false,
+  isEmailValidated: retrievedProfilesWithPdvId.isEmailValidated ?? false,
+  isInboxEnabled: retrievedProfilesWithPdvId.isEmailEnabled ?? false,
+  isWebhookEnabled: retrievedProfilesWithPdvId.isWebhookEnabled ?? false,
+  lastAppVersion: retrievedProfilesWithPdvId.lastAppVersion ?? "UNSET",
```
[the remark this answers is not in the document]
Comment (BurnedMarshal) [reply utils/formatter/profilesAvroFormatter.ts]: Fixed in [b9ac886](https://github.com/pagopa/io-functions-elt/pull/84/commits/b9ac8862622acc1e7cdf57071bf61b0736d4ccd3)

File: businesslogic/__tests__/analytics-profiles.test.ts [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+  isTestProfile: false,
+  lastAppVersion: "10.1.1" as RetrievedProfile["lastAppVersion"],
+  _ts: 1637077231000
+};
+const aRetrievedProfileList = [
+  aRetrievedProfile,
+  {
+    ...aRetrievedProfile,
+    id: `${aFiscalCode}-0000000000000003` as NonEmptyString,
+    settingsVersion: 3 as number & INonNegativeIntegerTag,
```
Comment (arcogabbo) [inline businesslogic/__tests__/analytics-profiles.test.ts:72]: typo?
```suggestion
    version: 3 as NonNegativeInteger,
```

Comment (BurnedMarshal) [reply businesslogic/__tests__/analytics-profiles.test.ts]: Fixed in 86cbd75

Comment (arcogabbo) [reply businesslogic/__tests__/analytics-profiles.test.ts]: the name of the field needs to be changed too @BurnedMarshal

Comment (BurnedMarshal) [reply businesslogic/__tests__/analytics-profiles.test.ts]: 😶‍🌫️ you are right, fixed in f00df91

**Review by gquadrati — APPROVED** [review]
LGTM, let's test it before merge

**Pull request merged**